.github/workflows/main.yml

```diff
@@ -47,12 +48,13 @@ jobs:
       run: .github/bin/dist-github-release.sh
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-    - if: github.event_name != 'pull_request'
+    - if: github.event_name != 'pull_request' && env.NUGET_API_KEY != ''
```

Comment:
Does it fail without this change?  I believe the *dist-nuget.sh* script already has addressed this case:

https://github.com/planetarium/libplanet/blob/e65b5e7161478a393e115b8cbe60f6bfb91aa0e0/.github/bin/dist-nuget.sh#L25-L36

Reply:
I'm not sure but maybe this change might be better than following the below guide:
> It would be silent if you configure a secret named NUGET_API_KEY with an empty string in your fork repo.

Reply:
Fair enough.